docs/components/SimpleInputDocs.js

```diff
@@ -25,14 +25,40 @@ class SimpleInputDocs extends React.Component {
         />
 
         <h3>Usage</h3>
+        <h5>baseColor <label>String</label></h5>
+        <p>The color of the <Code>input</Code> border on focus.</p>
+
         <h5>elementProps <label>Object</label></h5>
-        <p>Properties to pass directly to the <Code>input</Code> element.</p>
+        <p>Attributes to pass directly to the <Code>input</Code> element. ie. placeholder, value, onchange, etc.</p>
```

Comment:
I think I'd prefer the term props, or properties here rather than attributes - since we're still in React, these are technically props